### PR TITLE
Add get-device-login-history command to CS Falcon integration

### PR DESCRIPTION
**Pull request by ali-sawyer**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Add new command to the CrowdStrike Falcon integration – `cs-falcon-get-device-login-history` – to get recent user login history for a given device.

## Screenshots
Screen Shot 2022-10-15 at 4 22 39 PM

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py [hunk 1]
```diff
@@ -3806,6 +3814,33 @@ def get_detection_for_incident_command(incident_id: str) -> CommandResults:
                           raw_response=detection_res)
 
 
+def get_device_login_history_command(args):
+    device_id = args.get('device_id')
+    response = get_device_login_history(device_id)
+    resources = response.get('resources')
```
Comment (esharf) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py:3820]:
```suggestion
    resources = response.get('resources', [])
```

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py [hunk 2]
```diff
@@ -3806,6 +3814,33 @@ def get_detection_for_incident_command(incident_id: str) -> CommandResults:
...
+
+    command_results = []
+    for resource in resources:
+        logins_list = []
+        for login in resource.get('recent_logins'):
```
Comment (esharf) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py:3825]:
```suggestion
        for login in resource.get('recent_logins', []):
```

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml [hunk 3]
```diff
@@ -83,13 +83,13 @@ configuration:
...
   type: 16
-- defaultvalue: '1'
+- defaultvalue: "1"
   display: Incidents Fetch Interval
   name: incidentFetchInterval
   required: false
   type: 19
 description: The CrowdStrike Falcon OAuth 2 API (formerly the Falcon Firehose API), enables fetching and resolving detections, searching devices, getting behaviors by ID, containing hosts, and lifting host containment.
-display: CrowdStrike Falcon
+display: CrowdstrikeFalcon
```
Comment (esharf) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml:92]:
```suggestion
display: Crowdstrike Falcon
```

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md [hunk 4]
```diff
@@ -3925,7 +3925,7 @@ Gets the detections for a specific incident.
 | CrowdStrike.IncidentDetection.detection_ids | String | A list of detection ids connected to the incident. | 
 
 #### Command example
-```!cs-falcon-get-detections-for-incident incident_id=`inc:0bde2c4645294245aca522971ccc44c4:1a1eb17d1f9e4d82a9e8ba73d1095593````
+```!cs-falcon-get-detections-for-incident incident_id=`inc:0bde2c4645294245aca522971ccc44c4:1a1eb17d1f9e4d82a9e8ba73d1095593```
```
Comment (esharf) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md:3928]: nice catch

File: Packs/CrowdStrikeFalcon/TestPlaybooks/playbook-CrowdStrikeFalcon-Test.yml [hunk 5]
```diff
@@ -2624,8 +2684,43 @@ tasks:
...
+      script: '|||cs-falcon-get-device-login-history'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "22"
+    scriptarguments:
+      device_id:
+        simple: c1a4575cd124419db5ed3c1e8ddc2c04
```
Comment (esharf) [inline Packs/CrowdStrikeFalcon/TestPlaybooks/playbook-CrowdStrikeFalcon-Test.yml:2716]: you should add some method to get this id from the integration (so this playbook will work in every environment, not just in yours).

**Review by esharf — CHANGES_REQUESTED** [review]

**Comment by esharf** [comment]
@ali-sawyer did you managed to get any progress here? (if you need any help I’m available in [DFIR](https://dfircommunity.slack.com/team/UQWL4FQJ0))

**Review by esharf — APPROVED** [review]
will handle the comments here internally

**Comment by esharf** [comment]
@ali-sawyer i still need you to complete the [form](https://forms.gle/XDfxU4E61ZwEESSMA) please LMK once you are done with that.

**Comment by ali-sawyer** [comment]
@esharf done. I have been very busy with client work over the past couple months, so thank you very much for taking care of the changes and getting this merged!

**Pull request merged**